ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2

```diff
@@ -12,8 +12,10 @@
 
 | Type | Commands | Record type | Group | Logging |
 | ---- | -------- | ----------- | ----- | ------- |
-{%     if aaa_accounting.exec.console.type is arista.avd.defined and aaa_accounting.exec.console.group is arista.avd.defined %}
-| Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ aaa_accounting.exec.console.group }} | - |
+{%     if aaa_accounting.exec.console.type is arista.avd.defined  %}
+{%         set logging = aaa_accounting.exec.console.logging | arista.avd.default("False") %}
+{%         set group = aaa_accounting.exec.console.group | arista.avd.default(" - ") %}
+| Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ group }} | {{ logging }} |
```

Comment:
Aligning to our regular implementation and avoid overwriting common names outside of a loop.
```suggestion
{%         set aaa_accounting_logging = aaa_accounting.exec.console.logging | arista.avd.default("-") %}
{%         set aaa_accounting_group = aaa_accounting.exec.console.group | arista.avd.default("-") %}
| Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ aaa_accounting_group }} | {{ aaa_accounting_logging }} |
```
and please do the same below.